recipe/meta.yaml

```diff
@@ -14,16 +14,36 @@ build:
   number: 0
   skip: true  # [win]
 
-requirements:
-  build:
-    - {{ compiler('c') }}
-    - make
-
-test:
-  commands:
-    - ln --relative --help
-    - wc --version
-    - cat --version
+outputs:
+  
+  - name: {{ name }}
+    script: build_coreutils.sh
+
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - make
+
+    test:
+      commands:
+        - ln --relative --help
+        - wc --version
+        - cat --version
+
+  - name: gnu-{{ name }}
+    script: build_gnu-coreutils.sh
+
+
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - make
+
+    test:
+      commands:
+        - gln --relative --help
+        - gwc --version
+        - gcat --version
```

Comment:
Can we add a test that `ln` is not coming from this package?

Reply:
Sure, fwiw I tested exactly that locally 😄

Reply:
```

GNU coreutils online help: <https://www.gnu.org/software/coreutils/>
Full documentation <https://www.gnu.org/software/coreutils/ln>
or available locally via: info '(coreutils) ln invocation'
ln (GNU coreutils) 8.22
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Mike Parker and David MacKenzie.
ln (GNU coreutils) 9.0
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

```